.rubocop.yml

```diff
@@ -2,3 +2,11 @@
 # alphabetically
 inherit_from:
   - https://raw.githubusercontent.com/hanami/devtools/master/.rubocop.yml
+
+AllCops:
+  TargetRubyVersion: 2.3
+  Exclude:
+    - 'spec/support/fixtures/foo'
```

Comment:
Could we make the fixture adhere to the style guide instead of doing this?

Reply:
I am doing it now..